sdk/src/main/java/com/mapbox/maps/ResourceOptionsManager.kt

```diff
@@ -24,20 +26,21 @@ class ResourceOptionsManager(
    * Static variables and methods.
    */
   companion object {
-    private lateinit var default: ResourceOptionsManager
+    @VisibleForTesting(otherwise = PRIVATE)
+    internal var default: ResourceOptionsManager? = null
 
     /**
      * The default shared instance with default resource options.
      */
     @Synchronized
-    fun getDefault(context: Context): ResourceOptionsManager {
-      if (!this::default.isInitialized) {
```

Comment:
why not using lateinit var and use nullable variable instead?

Reply:
Lateinit will cache the default value and thus fail the test cases.

Reply:
We can overwrite the default ResourceOptionsManager#resourceOptions with default parameters after test.